docs/timelines.md

```diff
@@ -0,0 +1,133 @@
+
+![Timeline transport controls](/docs/images/Timeline-Timecode-Transport.png)
+
+- \{ |< \} sets the live timecode back to zero.
+- \{ |> \} starts playing from the start timecode.
+- \{ > \} and \{ || \} are play / pause controls.
+- \{ Stop \} stops playing and sets the live timecode back to zero.
+- \{ Record \} enters live record mode but doesn't start timecode playback.
+
+When you are linked to a timecode (except Internal Timecode), the first four buttons are replaced by the timecode source name and only the Record button is shown.
+
+![Timeline external timecode controls](/docs/images/Timeline-Timecode-ExternalTransport.png)
+
+If the timecode source is disabled in the **Timecode** menu, the time is shown in red.
+
+![Timeline disabled](/docs/images/Timeline-Timecode-Disabled.png)
+
+- If your timecode source starts at a big number, you can set a time offset to allow you to use zero-referenced times on your timeline.
+See [Time Offset](./timelines/timeline-options.md#time-options).
+
+The overview bar
+------------------
+
+The overview bar at the bottom of the screen gives you a view of the whole timeline from start to end. Triggers and markers are shown in miniature on the bar. You can make the main view zoom in and out by dragging the end markers of the bar to enclose a particular section. It's a good way of moving quickly around the timeline.
+
+![Timeline overview bar](/docs/images/Timeline-Overview-Bar-Cropped.png)
+
+Moving around the timeline view with the wheels
+-----------------
+
+Using the context menu option \{Timeline Wheels\} you can set the wheels to control your view.
+The wheel functions are
+- Wheel A: Horizontal scroll
+- Wheel B: Vertical scroll
+- Wheel C: Zoom
+
+Cursor
+-----------------
+
+The grey user cursor on the Timeline View sets the entry position for new triggers. It can also be used as a "play head" to 
+manually move the play position along the timeline while testing. You can position the cursor by tapping the screen in the ruler
+area at the top of the Timeline View.
+
+The \{Select Cursor\} context menu option sets Wheel A to control the cursor position.
+
+Selection tool types
+------------------
+
+The \{Arrow\} and \{Hand\} buttons let you choose different selection tools for editing.
+- \{Arrow\} is for marquee editing, where you select items by drawing a marquee selection box around them (or direct clicking).
+- \{Hand\} allows you to drag the timeline left and right or select by direct clicking.
+
+You can also select the tool using the \{Tool Pan\}/\{Tool Select\} context menu button.
+
+Table view
+------------------
+
+You can switch the Timeline View to a table version which shows details for each trigger in time order, in the same format as a cue list. Buttons down the left hand side allow you to filter which tracks are shown in the list.
```

Comment:
It's less switch the timeline view and more a totally separate view. You can have both on different screens at the same time for example. The selection is shared between the view